builder/struct.go

```diff
@@ -75,8 +75,9 @@ func (*Struct) Build(gen Generator, ctx *MethodContext, sourceID *xtype.JenID, s
 			continue
 		}
 
+		errWrapper := Wrap("error setting field %q", jen.Lit(targetField.Name()))
```

Comment:
The targetField name can be added directly to the string, no need for the placeholder.
```suggestion
		errWrapper := Wrap("error setting field " + targetField.Name())
```

Reply:
Golang will allocate the "error setting field %q" string only once even though it appears multiple times in code. 
I deliberately kept the field name as an argument to benefit from automatic const string interning in golang, see playground here: https://go.dev/play/p/3Rr6cYwTbjo. This way only the field names are new much smaller strings.

If I follow your advice, golang will allocate storage for all the strings unconditionally since they will be all new.

Reply:
Okay, this sounds reasonable.

Reply:
On the other thought, when dev sees an error "error setting field PostalCode" they will likely search the codebase for exact match to find out where it is coming from. Inlining the error message will make it easier for them - and since the whole purpose of this PR is to ease on troubleshooting, I have followed your advice to inline the error message in full. Please see my last iteration.